Header header
MetersCoordinates[] light
MetersCoordinates[] uncertainty_up
MetersCoordinates[] uncertainty_right
MetersCoordinates[] uncertainty_down
MetersCoordinates[] uncertainty_left
int32 lights_nr


